### ISSUE TITLE
Trying to set notification image in Thunderbird 129 throws an exception

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#258]
Trying to set notification image in Thunderbird 129 throws:

```
Uncaught (in promise) TypeError: notification.messageImage is undefined
    toggle_FilterMode chrome://quickfilters/content/qFilters-worker.js:142
    onMenuItemCommand chrome://quickfilters/content/quickFilters.js:864
    doCommand jar:file:///I:/Thunderbird_Profiles/ct4akusa.Beta-Test/extensions/quickFilters@axelg.com.xpi!/chrome/content/scripts/qFi-messenger.js:126
    oncommand chrome://messenger/content/messenger.xhtml:1
qFilters-worker.js:142:15

```

this can also have effects in the creation of filters - I am still gathering data this. JavaScript error logs would be very helpful in this case!!!

**Comment by RealRaven2000** [comment]
Here is an interim Version that tries to deal with the async change:

[quickFilters-wx-6.5.2pre22.zip](https://github.com/user-attachments/files/16487877/quickFilters-wx-6.5.2pre22.zip)

----

To try the version above download zip file and drag the file into Thunderbird Add-ons Manager (do not extract contents, it won't install like that)

**Comment by wa4zlw** [comment]
my issue was when you dragged a message to a folder the dialog box for the folder actions never showed up

this was fixed with the beta

leon

**Comment by RealRaven2000** [comment]
> my issue was when you dragged a message to a folder the dialog box for the folder actions never showed up
> 
> this was fixed with the beta
> 
> leon

I think it should be integrated in quickFIlters v [6.5.2](https://github.com/RealRaven2000/quickFilters/releases/tag/6.5.2) which was published yesterday. It was triggered by Thunderbird changing the notification function to be async.

**Comment by RealRaven2000** [comment]
Fixed in 6.5.2 - Published 09/08/2024
